prime_server/http_protocol.hpp

```diff
@@ -190,4 +194,19 @@ struct http_response_t : public http_entity_t {
 
 using http_server_t = server_t<http_request_t, http_request_info_t>;
 
+constexpr uint8_t ALL_VERBS_MASK = OPTIONS | GET | HEAD | POST | PUT | DELETE | TRACE | CONNECT;
+
+shortcircuiter_t<http_request_t> make_shortcircuiter(const std::string& health_check_path  = "/health_check",
+                                                     const uint8_t& verb_mask = ALL_VERBS_MASK,
+                                                     const std::string& allowed_origins = "*",
+                                                     const std::string& allowed_headers = "*",
+                                                     const int max_age = 86400);
```

Comment:
im still not a huge fan of all this configuration, especially cache headers but i do understand the desire to be complete.. i want to be sure that by default we get the same behavior as before as well. give me a day to make a better suggestion here (might just be changes to the default parameters or the way we send them to the function.. im not sure yet)

Reply:
I understand your concerns. To better manage configurations, we could encapsulate them in a struct. Let me know your thoughts or other suggestions.